tests/instancetype/instancetype.go

```diff
@@ -541,17 +541,18 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			vm := libvmi.NewVirtualMachine(vmi,
 				libvmi.WithInstancetype(instancetype.Name),
 				libvmi.WithPreference(preference.Name),
+				libvmi.WithRunStrategy(virtv1.RunStrategyAlways),
```

Comment:
While this isn't explicitly stated in the description of the test we would previously assert the behaviour of instance types and preferences against a stopped VM here. Switching this over to `Always` running changes that and drops coverage that I'd rather we keep.